layouts/shortcodes/spotlight-author.html

```diff
@@ -7,7 +7,7 @@
 <div class="author-container">
   <div class="author-details">{{- $details -}}</div>
   <div>
-      <span class="author-details">Written by: </span>{{- $author -}}</span>
+      <span class="author-details">Written by: {{- $author -}}</span>
```

Comment:
It looks like this styles the `$author` with `author-details`
```suggestion
      <span class="author-details">Written by: </span>{{- $author -}}
```

Before:
![image](https://github.com/cloudflare/cloudflare-docs/assets/94662631/d3241bd0-939e-4d86-96e1-5e45e86d5889)

After:
![image](https://github.com/cloudflare/cloudflare-docs/assets/94662631/6cdf4708-8769-40a9-8b3c-6e73ac66ac4c)

Reply:
Good catch, thanks